pacman-mirrors/PKGBUILD

```diff
@@ -1,8 +1,8 @@
 # Maintainer: Ray Donnelly <mingwandroid@gmail.com>
 
 pkgname=pacman-mirrors
-pkgver=20180604
-pkgrel=2
+pkgver=20200229
+pkgrel=3
```

Comment:
pkgrel=1

Reply:
got it and done, thank you for your support.